src/runtime/hvx_128.ll

```diff
@@ -64,6 +66,31 @@ define weak_odr <256 x i8> @halide.hexagon.deinterleave.vb(<256 x i8> %arg) noun
   ret <256 x i8> %r
 }
 
+define weak_odr <128 x i32> @halide.hexagon.deinterleave.vw.dv(<128 x i32> %arg) nounwind uwtable readnone alwaysinline {
```

Comment:
Do you really need to implement this? I think CodeGen_LLVM::call_intrin would do the right thing (call halide.hexagon.deinterleave.vw twice, and concat the results).